Petits-debuts-ui/src/Components/Trending/Trending.jsx

```diff
@@ -232,23 +228,29 @@ export default function Trending({
               value.cart +
               calculateOtherScore(value.others);
             //logic for exceptions
+            //typically each product is rated on a score out of 120
+            //but an absence of one or more pillar can reduce the total score to as low as 80
+            //we have to scale up to 120 regardless because our threshold mark is 70
             if (
               Object.keys(personalCart).length === 0 &&
               orderContext.length !== 0
             ) {
+              //this part scales up the score to 120 from 80
               score = (score * 120) / 95;
```

Comment:
It's be great to define what 120, 95, 105 is. They are kind of arbirary

Reply:
Those are the highest attatinable scores before scaling

i referenced their meaning a few lines above. do you want me to dig deeper?

Reply:
Define them to be also constants of what they mean.

MAXIMUM_SCORE / WHATEVER_95_MEANS

Reply:
okay, thanks